packages/admin-client/src/Applications/ApplicationList.tsx

```diff
@@ -0,0 +1,112 @@
+import { Box, Chip } from "@mui/material"
+import { FunctionField, Identifier, List, TextField, useUnselectAll } from "react-admin"
+import { DatagridStyles } from "../common/styles/DatagridStyles"
+import { useContext, useEffect } from "react"
+import CatchmentLabel from "../common/components/CatchmentLabel/CatchmentLabel"
+import { CatchmentContext } from "../common/contexts/CatchmentContext/CatchmentContext"
+import CustomDatagrid from "../common/components/CustomDatagrid/CustomDatagrid"
+import { FormBulkActionButtons } from "../common/components/FormBulkActionButtons/FormBulkActionButtons"
+import { ListActions } from "../common/components/ListActions/ListActions"
+import { ListAside } from "../common/components/ListAside/ListAside"
+
+export const applicationStatusFilters = {
+    All: { label: "All" },
+    New: { label: "New", status: "Submitted" },
+    InProgress: { label: "In Progress", status: "Processing" },
+    Completed: { label: "Completed", status: "Completed" },
+    Cancelled: { label: "Cancelled", status: "Cancelled" }
+} as { [key: string]: any }
+
+export const ApplicationList = (props: any) => {
+    const cc = useContext(CatchmentContext)
+    const unselectAll = useUnselectAll("applications")
+
+    useEffect(() => {
+        unselectAll()
+    }, [cc.catchment])
+
+    const handleRowClick = (id: Identifier, resource: string, record: any) => {
+        console.log("row click")
```

Comment:
Should remove the console logs and put the commented out temporary click functionality back in